dosenet.sh

```diff
@@ -23,13 +23,17 @@ PUBLICKEY=$CONFIGDIR/id_rsa_lbl.pub
 case "$1" in
   start)
     logger --stderr --id --tag $LOGTAG "Starting DoseNet script"
```

Comment:
Put this `logger` command after `ntp-wait`, but put a new logger command up here for "Waiting for NTP to be synced..."